scripts/tools/metricProgrammable.yml

```diff
@@ -476,7 +476,7 @@ returns:
         - "If trying to destroy a pre-defined metric group"
 --- #--------------------------------------------------------------------------
 type: function
-desc: "Destroy a created metric."
+desc: "Destroy a metric created using $tMetricCreateFromProgrammableExp."
```

Comment:
I think you had mentioned in an earlier comment you intended destroy to be applicable for all created metrics. but here you have updated desc to say this is applicable only for those created from ProgrammableExp. If so, I think it would be better to rename destroy API to reflect that

Reply:
>I think you had mentioned in an earlier comment you intended destroy to be applicable for all created metrics.

You are correct. This API is meant to destroy all created metrics.
I had updated the description here, since we have only zetMetricCreateFromProgrammableExp to create metrics now.
In the future, when we have more apis to create metrics, I thought we could update the description accordingly.

However, if it is confusing, I can rever the change

Reply:
ok sounds good, latest change looks fine by me